net-im/telegram-desktop-bin/telegram-desktop-bin-1.5.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils xdg versionator
```

Comment:
use `eapi7-ver`

Reply:
7 wont work because of the xdg class

Reply:
> 7 wont work because of the xdg class

I sis not mean `EAPI=7` but `eapi7-ver eclass`